androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/triage/history/HealthHistory.kt

```diff
@@ -0,0 +1,179 @@
+package com.EENX15_22_17.digital_journal.android.ui.triage.history
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.EENX15_22_17.digital_journal.android.dataModel.NursesNeeds
+import com.EENX15_22_17.digital_journal.android.dataModel.YesNo
+import com.EENX15_22_17.digital_journal.android.dataModel.nursesNeeds
+import com.EENX15_22_17.digital_journal.android.dataModel.yesNoLabels
+import com.EENX15_22_17.digital_journal.android.ui.theme.borderColor
+
+@Composable
+fun HealthHistoryPage(
+    healthHistoryViewModel: HealthHistoryViewModel = HealthHistoryViewModel(),
+    visitId: String = "id",
+    navBack: () -> Unit = { }
+) {
+    val rowModifier: Modifier = Modifier
+        .fillMaxWidth()
+        .padding(vertical = 6.dp, horizontal = 4.dp)
+        .border(
+            border = BorderStroke(
+                2.dp,
+                borderColor
+            ),
+            shape = RoundedCornerShape(8.dp)
+        )
+        .padding(horizontal = 16.dp, vertical = 12.dp)
+
+
+    Column(modifier = Modifier.fillMaxWidth()) {
+        Text(text = "Ankomst sidan för patient: $visitId")
+        Button(onClick = navBack) {
+            Text(text = "Gå tillbaka")
+        }
+        Row(modifier = rowModifier) {
+            healthHistoryViewModel.somaticHealth.value?.let { somaticHealth ->
+                SomaticHealth(
+                    choices = somaticHealthValues.keys.toTypedArray(),
+                    onChange = { healthHistoryViewModel.somaticHealth.value = it },
+                    currentSelected = healthHistoryViewModel.somaticHealth.value ?: emptySet(),
+                    labels = somaticHealthValues
+                )
+            }
+        }
+        Row {
+            Box(
```

Comment:
These boxes really clutter up the page, I think we should refactor them into a Group-component that has a consistent style for the whole app. This would reduce code duplication and also make the code more idiomatic.
However, this can also be done at a later stage in a separate PR.

Reply:
Agree. On Georges branch there is a component for this